### PR TITLE
Automatically import newly-created dataset

### PR DESCRIPTION
**Pull request by GrantPerkins**
Adds dataset to the navigation drawer. Still has download button for usability purposes.

### CODE REVIEW

**Review by AustinShalit — COMMENTED** [review]
Looks good. Because we don't have persistent data from the creation script anymore, can we move the "create" tmp directory to be inside of the container? That way it will clean itself up automatically if interrupted or something bad happens.

**Comment by GrantPerkins** [comment]
The only thing about that is there is a 2 gigabyte CSV that is downloaded, that I would like cached.

File: packages/server/src/datasources/dataset-service.ts [hunk 1]
```diff
@@ -86,10 +85,18 @@ export class DatasetService extends DataSource {
...
-    const id = uuidv4();
-    const directory = `data/create/${id}`;
+    const dataset = Dataset.build({ name: classes[0] });
+    const directory = `data/create/${dataset.id}`;
     await mkdirp(directory);
-    return await this.mLService.create(classes, maxImages, directory, id);
+    await mkdirp(`data/datasets/${dataset.id}`);
+    const createJob = await this.mLService.create(classes, maxImages, directory, dataset.id);
+    const name = createJob.zipPath.split("/")[1]; // Axon_Dataset_ETC.zip
+    dataset.name = name; // Axon_Dataset_ETC.zip
```
Comment (AustinShalit) [inline packages/server/src/datasources/dataset-service.ts]: Playing with this more and more, I think having the dataset name be something more like OpenImages_classname might be better?

Comment (GrantPerkins) [reply packages/server/src/datasources/dataset-service.ts]: `OpenImages_{class name}.zip` is now the .zip name.

**Comment by GrantPerkins** [comment]
This new commit only works with the fix from #232 btw @AustinShalit. They should be merged around the same time.

File: packages/server/src/mL/index.ts [hunk 2]
```diff
@@ -125,14 +125,16 @@ export default class MLService {
...
+  async create(classes: string[], maxImages: number, id: string): Promise<CreateJob> {
     const creator: Creator = new Creator(classes, maxImages, id);
 
     const validLabels = await creator.checkLabels();
     if (!validLabels.success) {
       return validLabels;
     }
+    console.log("Creating parameter file");
     await creator.writeParameterFile();
+    console.log("Startin Python script");
```
Comment (AustinShalit) [inline packages/server/src/mL/index.ts]:
```suggestion
    console.log("Starting Python script");
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**